components/screens/KNSAsset.tsx

```diff
@@ -0,0 +1,146 @@
+import { useParams } from "react-router-dom";
+import Header from "@/components/GeneralHeader";
+import patchCheckFill from "@/assets/images/patch-check-fill.png";
+import avatarIcon from "@/assets/images/avatar.png";
+import { useKns, AssetDataWithId } from "@/hooks/useKns.ts";
+import { useState, useRef } from "react";
+import { walletAddressEllipsis } from "@/lib/utils";
+import Copy from "@/components/Copy";
+import HoverShowAllCopy from "@/components/HoverShowAllCopy";
+import HoverShowAll from "@/components/HoverShowAll";
+
+export default function KNSAsset() {
+  const { assetId } = useParams();
+  const { fetchDomainDetails } = useKns();
```

Comment:
I think we should use `useSWR` for this one. I used fetch because I needed to use this method in `form` validation :/

Reply:
https://github.com/forbole/kastle/pull/58/commits/afb0d6fa3257695999c5050f5b7a22dc8cbe45ad